resources/views/members/search.blade.php

```diff
@@ -60,8 +60,7 @@
 	<tbody>
 		@forelse ($members as $member)
 		<tr>
-			<td><a href="{{ url('/members', $member->id) }}">{{$member->voornaam}} {{$member->tussenvoegsel}}
-					{{$member->achternaam}}</a></td>
```

Comment:
Why this change?
Having only the first name isn't that clear since we have 3 Davids. Maybe include the email?

Reply:
Changed it to `volnaam` which is the full name...